CMakeLists.txt

```diff
@@ -11,6 +11,31 @@ set(TESTS_SUFFIX $ENV{TESTS_SUFFIX})
 message(STATUS "Setting TESTS_SUFFIX on performance tests to '${TESTS_SUFFIX}'")
 set(ENV{BETTER_EXCEPTIONS} 1)
 
+find_package(Git)
+if(GIT_FOUND)
```

Comment:
That's not great for someone checking out the tarballs, which is by far the most likely scenario for actual releases.

Reply:
That's a good point. I've removed this, and we now just retrieve the version from the `PROJECT_VERSION` CMake variable, set [with this](https://cmake.org/cmake/help/latest/command/project.html#options).